### ISSUE TITLE
Is there a validation mechanism in the code ?

### ISSUE THREAD

**Issue by howardhsu** [songyouwei/ABSA-PyTorch#48]
Thanks for sharing and making all ABSA baselines together. 
Is there a validation mechanism in the code? I am not sure how to know which model (selection) to test.

**Comment by songyouwei** [comment]
What do you mean by ` validation mechanism` ? 
It's often assumed that the model with best results on testset correspond to the most appropriate model.

**Comment by GeneZC** [comment]
@songyouwei Or we could firstly do cross validation to select the best-performing model and do the experiments on test dataset with the selected one for better reproducibility.

**Comment by howardhsu** [comment]
Yes, that's what I meant. Either cross-validation on multiple folds or just a single validation set within the current training set should be fine to find the best model then go for testing only once. The best model picked from the multiple testing may already know the testing set well and could be lower in real-world testing.

**Comment by songyouwei** [comment]
We can add cross-validation as mentioned in #32 , or we can split an unofficial validation set from the existing test set.

**Comment by songyouwei** [comment]
#53
